### DISCUSSION TITLE
Any way to get a CI release back after it rolls out of web flasher?

### DISCUSSION THREAD

**Discussion by DangRock** [forkineye/ESPixelStick#807] [General]
Hello!  I've been successfully using the 8/17 CI release via the web flasher on all my ESPixelSticks and DigQuads. I see today it has rolled out of the web flasher due to a new CI build. Is there any way to get it back, or get a download of it with the flashing tool?  It's working well and the ones after it have had some choppy animation issues. I like to have everything on the same build and right now I have 15 sticks and 4 dig quads. Looking to add a couple more sticks and a dig quad. 

![image](https://github.com/user-attachments/assets/b5e3fb06-296c-4f3f-9c18-0efbe1b86ef7)

Thanks!

**Comment by joe8012** [comment]
I use this version too.  Here's a link to the zip in my google drive.  It's marked shared so you should be able to download it.  https://drive.google.com/file/d/1zjiD-A84xEDK8fqCWhS7xul0yivezALq/view?usp=sharing

**Comment by DangRock** [comment]
This is perfect. Thanks!
